DEVELOPING.md

```diff
@@ -113,18 +121,6 @@ Check if format is correct with `ktlint` or `./gradlew ktlint`.
 Run `ktlintFormat` or `./gradlew ktlintFormat` or `make fix` to automatically reformat everything and 
 fix all the issues.
 
-There's a script that installs pre-commit git hook to check `ktlint` on every commit :
-[install-pre-commit-hook](https://github.com/mapbox/mapbox-maps-android-internal/blob/main/scripts/hooks/install-pre-commit.sh). 
-Note, that if some global pre-commit (eg [secret-shield](https://github.com/mapbox/secret-shield)) 
-git hooks are already installed it will probably prevent this new hook from being called. 
-In such case global git hooks will need to call local hooks as well manually, e.g. add
-```
-if [ -e ./.git/hooks/pre-commit ]; then
-    ./.git/hooks/pre-commit "$@"
-fi
-```
-to the end of global pre-commit hook.
```

Comment:
removed since the scripts is pointing to an internal repository, have moved this section to the internal repo as well.